.travis.yml

```diff
@@ -1,11 +1,22 @@
 ---
 language: node_js
+node_js:
+  - "4"
+  - "6"
+  - "stable"
 
 sudo: false
 
 cache:
-  directories:
-    - node_modules
+  yarn: true
+
+before_script:
+  # ensure we are testing against latest versions of all deps when using Yarn
+  - yarn upgrade
+
+branches:
+  only:
+    - master
```

Comment:
I think the move has happened long enough ago for you to be able to fork again :man_shrugging: